packages/block-library/src/post-comments/index.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Server-side rendering of the `core/post-comments` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/post-comments` block on the server.
+ *
+ * @return string Returns the filtered post comments for the current post wrapped inside "p" tags.
+ */
+function render_block_core_post_comments() {
+	$post = gutenberg_get_post_from_context();
+	if ( ! $post ) {
+		return '';
+	}
+	$comments = get_comments(
+		array(
+			'post_id' => $post->ID,
+		)
+	);
+	$output   = '';
+	foreach ( $comments as $comment ) {
```

Comment:
We render the comments in a linear way without being possible to identify a comment is a reply to another comment, is this something we should handle now or the plan is to deal with this in follow up PR's?

Reply:
I think it'll be better if we address this in a polishing PR after we have some initial designs. This was just to get the general structure going.